tests/Aspire.StackExchange.Redis.Tests/ConformanceTests.cs

```diff
@@ -39,7 +39,7 @@ public class ConformanceTests : ConformanceTests<IConnectionMultiplexer, StackEx
                 "ConfigurationOptions": {
                   "CheckCertificateRevocation": true,
                   "ConnectTimeout": 5,
-                  "HeartbeatInterval": "PT5S",
+                  "HeartbeatInterval": "00:00:02",
```

Comment:
Will this change get exercised in a test? Were we not checking whether the value for `HeartbeatInterval` here could be correctly converted to a timespan, I mean, deserialize this json?

Reply:
This test ensures that this JSON string successfully validates against the JSON schema. It doesn't bind this JSON config to an actual object. We have other tests that do something like this here:

https://github.com/dotnet/aspire/blob/b18a246f2ba936997490c716264195715b2d8b1c/tests/Aspire.RabbitMQ.Client.Tests/AspireRabbitMQExtensionsTests.cs#L130-L160